unattended_scripts/open-distro/unattended-installation/unattended-installation.sh

```diff
@@ -136,6 +136,15 @@ checkArch() {
     
 }
 
+applyLog4j2Mitigation(){
+
+    mkdir /etc/elasticsearch/jvm.options.d
```

Comment:
Please use eval like on the rest of the script

Reply:
Done: 15c60fc9fea39cc957e4f480e8bf0ee6226570c2